pandas/core/arrays/datetimelike.py

```diff
@@ -1435,6 +1435,50 @@ def max(self, axis=None, skipna=True, *args, **kwargs):
         # Don't have to worry about NA `result`, since no NA went in.
         return self._box_func(result)
 
+    def mean(self, axis=None, skipna=True):
+        """
+        Return the mean value of the Array or mean along an axis.
+
+        .. versionadded:: 0.25.0
+
+        Parameters
+        ----------
+        axis : None
+            Dummy parameter to match NumPy signature
```

Comment:
If it is to match numpy's signature, then there are a bunch of other kwargs to add as well

Reply:
right and we do this elsewhere

Reply:
would just add kwargs for this purpose; we don't add them explicity anywhere else